canonical-web-and-design/maas-ui
# Make the basename optional.

Done:
- Make the basename optional and fully configurable. Fixes: #32.

QA:
- Change the `REACT_APP_BASENAME` in our `.env.development.local` to whatever you like
- `./run`
- Your app should load at that URL.
